### ISSUE TITLE
The 'psutil>=5.2.2' distribution was not found

### ISSUE THREAD

**Issue by yveskerwyn** [Jumpscale/ays9#399]
#### Installation information

- jumpscale version: 9.2.1
- operating system: Ubuntu

#### Detailed description

After having installed an AYS server I had to manually install psutil==5.2.2.

#### Relevant stacktraces

```
Traceback (most recent call last):
  File "/usr/local/bin/ays", line 4, in <module>
    __import__('pkg_resources').require('JumpScale9AYS==9.2.0')
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 3144, in <module>
    @_call_aside
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 3128, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 3157, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 668, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 681, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 870, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'psutil>=5.2.2' distribution was not found and is required by JumpScale9Lib
```

**Comment by ahussein** [comment]
@yveskerwyn Can you please add how did you install the AYS server ?

**Comment by yveskerwyn** [comment]
@abdulrahmantkhalifa 

This is how I installed the AYS server: https://github.com/yveskerwyn/jumpscale/blob/master/2-remote_ays_setup.md

With every installation of AYS server I need to execute `pip install psutil==5.2.2`

**Comment by yveskerwyn** [comment]
Can this be fixed in 9.2.1 please

Is really easy to reproduce, just install AYS in a new VM

**Comment by hossamabdullah** [comment]
Now to avoid having this we will need to add a value to the full parameter
ays_host.prefab.js9.js9core.install(branch=branch, full_installation=True)
